types/ip-regex/ip-regex-tests.ts

```diff
@@ -1,7 +1,13 @@
-import ipRegex = require('ip-regex');
+import * as ipRegex from 'ip-regex';
```

Comment:
the sources uses commonjs exports, so `export = ipRegex` and `import ipRegex = require('ip-regex')` is correct. Please revert this line.